cypress/e2e/wip/dbot/pageobjects/common.js

```diff
@@ -111,6 +111,10 @@ class Common {
     this.blocklyDurationValue.type('2')
   }
 
+  blockDashboardLoad = () => {
+    cy.get('.bot-dashboard.bot').should('be.visible').wait(2000)
```

Comment:
Please use dynamic wait.
```suggestion
    cy.get('.bot-dashboard.bot', {timeout:2000}).should('be.visible')
```

Reply:
Also, just remembered. Cypress default timeout is already 4 sec so this isn't even needed unless you wait for something greater than 4 sec. ⏲️

Reply:
ok will remove the wait then